C/Tarefa/2/sensors.c

```diff
@@ -15,8 +16,11 @@ int get_value() {
 }
 
 void send_data(uint8_t data) {
-    // Desfaçam o empacotamento e imprimam
-
-    for (int i = 0; i < 8; i++)
-        printf("Sensor %d: %d", i, );
+    int k=1, i=0; //cria dois contadores
+	
+    for (i=7; i>=0; i--){
+	printf ("Sensor %d: %d\n", k, (data&1)); //k imprime o número do sensor, enquanto data&1 utiliza o operador lógico and para verificar se o bit mais significativo de data é 1, caso contrário, retornará 0. A ideia é de que, na utilização em um robô, bastaria enviar o valor de data&1 ao invés de utilizar o printf.

```

Comment:
O bit mais significativo é o mais à esquerda, então nesse caso você verificaria o menos significativo.
Quando for fazer comentários grandes, é melhor separar em mais de uma linha pra ficar mais agradável pra ler. Pode usar ``/* (linhas de comentário) */ `` pra não ficar com ``//`` no começo de todas as linhas.